src/components/filters/__tests__/input.test.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+import { fireEvent, render } from "@testing-library/react"
+
+import InputFilter from "../input"
+
+describe("<InputFilter>", () => {
+  describe("renders correctly", () => {
+    it("with no values", () => {
+      const { container } = render(
+        <InputFilter
+          mode="numeric"
+          name="test"
+          placeholder="test"
+          apply={jest.fn()}
+        />
+      )
+
+      expect(container).toMatchSnapshot()
+    })
+
+    it("with autofocus after focusing", () => {
+      const { container, getByPlaceholderText } = render(
+        <InputFilter
+          mode="numeric"
+          name="test"
+          placeholder="test"
+          apply={jest.fn()}
+        />
+      )
+
+      fireEvent.focus(getByPlaceholderText("test"))
+
+      expect(container).toMatchSnapshot()
```

Comment:
how is focussing changing the resulting snapshot? is there something specific we can expect instead of taking a snapshot?

Reply:
I believe it adds autofocus prop. Maybe @lkappeler can confirm that? I did copy over snapshots from listings...